docs/serving/README.md

```diff
@@ -1,16 +1,14 @@
 # Knative Serving
 
-Knative Serving builds on Kubernetes to support deploying and serving
-of serverless applications and functions. Serving is easy to get started with
```

Comment:
The second sentence is definitely fluff, but I feel like we should keep the first sentence (or something similar). I like having a sentence introducing what Knative is, especially for new users who won't already know this. Other than that, LGTM.